ui/raidboss/data/05-shb/raid/e9n.js

```diff
@@ -222,23 +222,92 @@ export default {
   timelineReplace: [
     {
       'locale': 'de',
-      'missingTranslations': true,
       'replaceSync': {
         'Cloud Of Darkness': 'Wolke der Dunkelheit',
+        'Stygian Arbor': 'stygisch Dorn',
```

Comment:
to be safe i would write it as `'Stygian Arbor': 'stygisch(?:e|er|es|en) Dorn',`

Reply:
Thanks!